src/Assets/TestProjects/PackageValidationTestProject/PackageValidationTestProject.csproj

```diff
@@ -1,8 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <Sdk Name="Microsoft.DotNet.PackageValidation" Version="1.0.0-test" />
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
     <DefineConstants Condition="'$(ForceValidationProblem)' == 'true'">$(DefineConstants);ForceValidationProblem</DefineConstants>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MSBuildProjectDirectory)\..\..\..\..\src\Compatibility\Microsoft.DotNet.PackageValidation\Microsoft.DotNet.PackageValidation.csproj" />
+    <PackageReference Include="NuGet.Frameworks" Version="6.0.0-preview.1.66" />
+    <PackageReference Include="NuGet.Packaging" Version="6.0.0-preview.1.66" />
+    <PackageReference Include="Microsoft.Bcl.HashCode" Version="1.1.1" />
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="4.0.0-1.21277.15" />
+  </ItemGroup>
+
+  <Import Project="$(MSBuildProjectDirectory)\..\..\..\..\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets" />
+  
+  <PropertyGroup>
+    <DotNetPackageValidationAssembly Condition="'$(MSBuildRuntimeType)' != 'Core'">$(MSBuildThisFileDirectory)\bin\$(Configuration)\netstandard2.0\Microsoft.DotNet.PackageValidation.dll</DotNetPackageValidationAssembly>
```

Comment:
Instead of hardcoding paths that contain tfms (that could change), you could instead figure that out at runtime in the test setup and pass that as a property globally?

Reply:
These paths depend on the test project and not on Microsoft.Dotnet.PackageValidation tfms

Reply:
Actually could you use OutDir or TargetPath to figure out this path?

Reply:
pack runs as  part of outerbuild so both of these props are not set.

Reply:
What about `BaseOutputPath` then?

Reply:
```BaseOutputPath``` is just bin and output path is ```bin/$(configuration)```. we still have to specify the tfm.

Reply:
I think it is still worth to use `OutputPath` and then just hardcode the tfm? That way we avoid hardcoding configuration?

Reply:
I forget these props are set in .targets file i.e after the csproj so we wont be able to use them

Reply:
Ok, then I guess we can leave with this on the tests.